src/registrar/templates/application_review.html

```diff
@@ -5,6 +5,18 @@
 {# there are no required fields on this page so don't show this #}
 {% endblock %}
 
+{% block form_page_title %}
+  <span class="display-flex flex-align-start" > 
+    <img 
+        class= "margin-right-105" 
+        src="{%static 'img/registrar/dotgov_review_magnify.svg' %}" 
+        alt=""
+        width="72"
+        />
+    <h1> Review and submit your domain&nbsp;request </h1>
```

Comment:
The non-breaking space is causing the layout to look a bit wonky on smaller screens:

![image](https://user-images.githubusercontent.com/15718530/228907977-a0564093-26ba-4c8a-a7ba-ab41d5b901c5.png)

Reply:
Thanks Seamus! Tried a different approach with a title instead